### PR TITLE
WIP - Unify Kubernetes example

### PR DESCRIPTION
**Pull request by johscheuer**
@sttts @quolix

I tried to unify the two Kubernetes setups in this repo. Also this solves https://github.com/quobyte/quobyte-deploy/issues/3 + https://github.com/quobyte/quobyte-deploy/issues/6

I tested this setup with Kubernetes 1.3.6 + CoreOS and Kubernetes 1.4 + CentOS 7 (bare-metal)

There are some steps to resolve to merge this in the repo but feel free to comment :)

# TODO

- [X] Check Availability (webconsole) from outside (proxy/port-foward works)
- [X] Make Quobyte Api externally available
- [X] Add Readme how-to access Webconsole + API
- [X] Adjust Readme
- [X] Registry bootstrap delete device and decommission
- [X] Get rid-off Bootstrap Pod -> DS if no srv entry make bootstrap (only one node is labled as `registry`)
- [x] Deregister (Registry) services
- [x] Check if PetSet is better than daemonset -> stable names (?)
- [x] Use official Quobyte Plugin with Quobyte on Kubernetes (Testing + K8s 1.4+)
- [X] Make client-ds work (use util-linux ns-enter) -> nsenter is broken (<https://github.com/karelzak/util-linux/issues/315>) -> Todo for Quobyte (1.3.6)
- [ ] Bootstraping Tool for Quobyte (complete automated setup)

### CODE REVIEW

File: kubernetes/README.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+```bash
+$ kubectl create -f qmgmt-pod.yaml
+$ kubectl exec -it qmgmt-pod -- qmgmt -u api:7860 device list
+$ kubectl exec -it qmgmt-pod -- qmgmt -u api:7860 registry list
+```
+
+After this step you can remove the bootstrap pod if you want.
+
+```bash
+$ kubectl delete -f  registry-bootstrap-pod.yaml
```
Comment (sttts) [inline kubernetes/README.md]: Don't see a create call for registry-bootstrap-pod.yaml anymore.

File: kubernetes/README.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+
+```bash
+$ kubectl logs example -f
+Found old state starting at Tue May 10 10:01:39 UTC 2016
+Tue May 10 10:07:32 UTC 2016
+Tue May 10 10:07:37 UTC 2016
+Tue May 10 10:07:42 UTC 2016
+```
+
+## Acces Webconsole and API
```
Comment (sttts) [inline kubernetes/README.md]: Access

File: kubernetes/example-pod.yaml [hunk 3]
```diff
@@ -20,6 +20,10 @@ spec:
           fi
           while sleep 5; do date | tee -a /persistent/log; done
   volumes:
-  - name: persistent
-    hostPath:
-      path: /storage/cluster/example
+  - name: quobytevolume
+    quobyte:
+      registry: LBIP:7861
```
Comment (sttts) [inline kubernetes/example-pod.yaml]: LBIP ?

File: kubernetes/qmgmt-pod.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  name: qmgmt-pod
+spec:
+  containers:
+    - name: qmgmt-pod
+      image: quay.io/quobyte/quobyte-server:1.3.4
+      command:
+        - /bin/bash
+        - -xec
+        - |
+          while true; do sleep 90000; done
```
Comment (sttts) [inline kubernetes/qmgmt-pod.yaml]: tail -f /dev/null

Comment (johscheuer) [inline kubernetes/qmgmt-pod.yaml]: I thought of sleep to minimize the cpu usage but `tail -f /dev/null` seems to work too :)

**Pull request merged**